spark3/src/main/java/org/apache/iceberg/spark/source/SparkWrite.java

```diff
@@ -72,52 +75,67 @@
 import static org.apache.iceberg.TableProperties.WRITE_TARGET_FILE_SIZE_BYTES;
 import static org.apache.iceberg.TableProperties.WRITE_TARGET_FILE_SIZE_BYTES_DEFAULT;
 
-class SparkBatchWrite implements BatchWrite {
-  private static final Logger LOG = LoggerFactory.getLogger(SparkBatchWrite.class);
+class SparkWrite {
```

Comment:
In the future, Spark may have a write abstraction so we will make this class extend `Write`.